src/main/java/mate/academy/dao/jdbc/ManufacturerJdbcDao.java

```diff
@@ -0,0 +1,116 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ManufacturerDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerJdbcDao implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO `manufacturers` (`manufacturer_name`, `manufacturer_country`) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement
+                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet != null && resultSet.next()) {
```

Comment:
```suggestion
            if (resultSet.next()) {
```
resultSet won't be null, it can just be empty, so this check is redundant